fuzzers/forkserver/fuzzbench_forkserver_sand/src/lib.rs

```diff
@@ -1,9 +1,19 @@
-use libafl_targets::{map_shared_memory, start_forkserver};
+use libafl_targets::{
+    map_input_shared_memory, map_shared_memory, start_forkserver, MaybePersistentForkserverParent,
+};
 
 #[no_mangle]
 pub extern "C" fn libafl_start_forkserver() {
     // Map shared memory region for the edge coverage map
-    map_shared_memory();
+    if map_shared_memory().is_err() {
+        std::process::exit(1);
```

Comment:
Can we propagate this error like returning a Result? For previous code calling this we can write a wrapper version to exit(1). This way we can behave like aflpp style “run once if there is no aflpp”

Reply:
Oh I see, that is map_input_shared_memory right?

Reply:
Yes, the error can be propagated. However, this is just a fuzzer example, which is not intended to be used as a library. Where shall the exit-wrapper be placed? (In fact, now all related codes in libafl main crate now already no longer exit)

Reply:
Nah, I just notice it's a fuzzer example when I saw  `map_input_shared_memory` so it's pretty should be here =).